plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/TrinoGlueCatalog.java

```diff
@@ -1195,17 +1225,19 @@ public void updateMaterializedViewColumnComment(ConnectorSession session, Schema
                 definition.getPath(),
                 definition.getProperties());
 
-        updateMaterializedView(session, viewName, newDefinition);
+        updateMaterializedView(viewName, newDefinition);
     }
 
-    private void updateMaterializedView(ConnectorSession session, SchemaTableName viewName, ConnectorMaterializedViewDefinition newDefinition)
+    private void updateMaterializedView(SchemaTableName viewName, ConnectorMaterializedViewDefinition newDefinition)
     {
-        TableInput materializedViewTableInput = getMaterializedViewTableInput(
```

Comment:
Why this change?

Reply:
The new code copies the existing MV's table parameters while the old code blows them away. This is important since the paramaters include the storage table metadata path.

Reply:
if so, we could still use `getMaterializedViewTableInput`, just provide `getTableParameters(table)` as parameters, instead of `createMaterializedViewProperties`, right?

Reply:
Sure, that should work.

Reply:
like

```java
com.amazonaws.services.glue.model.Table table = getTable(viewName, false);
TableInput materializedViewTableInput = getMaterializedViewTableInput(
        viewName.getTableName(),
        encodeMaterializedViewData(fromConnectorMaterializedViewDefinition(newDefinition)),
        table.getOwner(),
        getTableParameters(table));
```